internal/strings/strings.go

```diff
@@ -0,0 +1,23 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package strings
+
+import "fmt"
+
+func ToString(v interface{}) string {
```

Comment:
nitpick: do we really use that so often than it warrants a function? and it' s own pkg

Reply:
there are a couple. 13 usages (excl. *_test.go).
It's likely that their might be other string manipulation functions somewhere lying around that could also go into the file now or in the future. So if ur ok i will keep it for now.

Reply:
:+1:  Didn't expect it's used that much, but agree in that case makes perfect sense to keep it arround